src/scripts/generate-test-patient-spreadsheets.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+set -euo pipefail
+if [ "${DEBUG:=false}" == "true" ]; then set -x; fi
+export WORK=$(mktemp -p /tmp -d work.XXXX)
+
+main() {
+  local outputXlsx="${2}" tmpOutputCsv
+  TEMPLATE_FILE="${1}"
+  tmpOutputCsv="${WORK}/$(basename "${outputXlsx}" | sed 's/\.xlsx$/.csv/')"
```

Comment:
Why do all this processing for a temp file when `mktemp` works all the same?

Reply:
Because libreoffice conversion does not allow us to specify an output filename (only output directory) so we need the initial CSV name to match what we want the resultant xlsx to be.